warehouse/query-core/src/test/java/datawave/query/QueryTestTableHelper.java

```diff
@@ -108,39 +115,33 @@ private void deleteAndCreateTable(TableOperations tops, String tableName) throws
         tops.create(tableName);
     }
     
-    public void setupWriters(MockAccumuloRecordWriter recordWriter) throws TableNotFoundException {
-        BatchWriterConfig bwCfg = new BatchWriterConfig().setMaxLatency(1, TimeUnit.SECONDS).setMaxMemory(1000L).setMaxWriteThreads(1);
-        recordWriter.addWriter(new Text(METADATA_TABLE_NAME), connector.createBatchWriter(METADATA_TABLE_NAME, bwCfg));
-        recordWriter.addWriter(new Text(DATE_INDEX_TABLE_NAME), connector.createBatchWriter(DATE_INDEX_TABLE_NAME, bwCfg));
-        recordWriter.addWriter(new Text(LOAD_DATES_METADATA_TABLE_NAME), connector.createBatchWriter(LOAD_DATES_METADATA_TABLE_NAME, bwCfg));
-        recordWriter.addWriter(new Text(SHARD_TABLE_NAME), connector.createBatchWriter(SHARD_TABLE_NAME, bwCfg));
-        recordWriter.addWriter(new Text(SHARD_INDEX_TABLE_NAME), connector.createBatchWriter(SHARD_INDEX_TABLE_NAME, bwCfg));
-        recordWriter.addWriter(new Text(SHARD_RINDEX_TABLE_NAME), connector.createBatchWriter(SHARD_RINDEX_TABLE_NAME, bwCfg));
-        recordWriter.addWriter(new Text(SHARD_DICT_INDEX_NAME), connector.createBatchWriter(SHARD_DICT_INDEX_NAME, bwCfg));
-        recordWriter.addWriter(new Text(MODEL_TABLE_NAME), connector.createBatchWriter(MODEL_TABLE_NAME, bwCfg));
-    }
-    
-    public void configureTables() throws AccumuloSecurityException, AccumuloException, TableNotFoundException {
-        configureShardIndex();
-        configureShardReverseIndex();
+    /**
+     * Configures all of the default tables and associates a {@link BatchWriterConfig} object for ach table.
+     * 
+     * @param writer
+     * @throws AccumuloSecurityException
+     * @throws AccumuloException
+     * @throws TableNotFoundException
+     */
+    public void configureTables(MockAccumuloRecordWriter writer) throws AccumuloSecurityException, AccumuloException, TableNotFoundException {
+        configureAShardRelatedTable(writer, new MetadataTableConfigHelper(), ShardedDataTypeHandler.METADATA_TABLE_NAME, METADATA_TABLE_NAME);
```

Comment:
What happened to the dateIndex table?  I am guessing we have no date index tests.

Reply:
I didn't see any date related tests in the RefactoredFunctionalQueryTest. Perhaps this is in another test class? It can be added later.